pom.xml

```
@@ -89,6 +89,7 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
     <org.apache.httpcomponents.httpclient.version>4.5.13</org.apache.httpcomponents.httpclient.version>
     <org.apache.poi.version>5.2.2</org.apache.poi.version>
     <org.aspectj.version>1.8.8</org.aspectj.version>
+    <org.bouncycastle.version>1.67</org.bouncycastle.version>
```

Currently package version is `1.68`. You can even use `1.70` (latest).

```suggestion
    <org.bouncycastle.version>1.68</org.bouncycastle.version>
```

---

Done.